scripts.d/support/servers.pvsc

```diff
@@ -12,13 +12,13 @@
           <String default="00:30:00"/>
         </Option>
         <Option label="Number of Nodes:" name="SLURM_NODES" save="true">
-          <Range default="4" max="64" min="1" step="1" type="int"/>
+          <Range default="2" max="2" min="1" step="1" type="int"/>
```

Comment:
The default for our case is 2 max, but is that really the case for all projects?

Reply:
I think, It's forbidden to use more than 2 for our group.
I don't know if it's the case in general.

Reply:
So better letting the 64 here. Some other WCKEY group may be allowed to use more nodes. So let's keep 2 as default and 64 as max.